tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -400,6 +400,7 @@ Restore PVC Value To Default Size
     Click Button    Restore Default
     Wait Until Keyword Succeeds    30    1
     ...    Wait Until Page Contains    Cluster settings updated successfully.
+    Sleep    120s   reason=Wait until JupyterHub leader pod is elected
```

Comment:
have a look at pr #208 it defines a keyword to handle Bad Gateway page after rolling out. `Handle Bad Gateway Page`.

Since #208  has not been merged yet, you could just copy the keyword here

Reply:
Another option is to run   `Verify JupyterHub Deployment` inside `Wait Until Keyword Succeeds `  similar as it's done [here](https://github.com/red-hat-data-services/ods-ci/blob/59a3ec8bdc75fcc84247bf2d147f7a0a0176679d/tests/Resources/Page/ODH/Prometheus/Prometheus.robot#L169)